clients/java/dkv-client/src/test/resources/dkv_config.yaml

```diff
@@ -6,7 +6,11 @@ block-cache-size : 3221225472
 root-folder : "/tmp/dkvsrv"
 dc-id : "default"
 disable-auto-master-disc : false
-discovery-service-config : "internal/discovery/discovery.ini"
+status-ttl           :   300
+heartbeat-timeout    :   60
+discovery-service-addr    :   "multi:///127.0.0.1:8001,127.0.0.1:8002,127.0.0.1:8003"
+push-status-interval      :   1
+poll-cluster-info-interval :   1
```

Comment:
It would make to not have this at top level but under a section. Something like

```yaml
discovery-service: 
  status-ttl           :   300
  heartbeat-timeout    :   60
  discovery-service-addr    :   "multi:///127.0.0.1:8001,127.0.0.1:8002,127.0.0.1:8003"
  push-status-interval      :   1
  poll-cluster-info-interval :   1
```